services/marketplace/client.go

```diff
@@ -78,6 +88,29 @@ func (c *Client) GetPlugin(filter *model.MarketplacePluginFilter, pluginVersion
 	return nil, errors.New("plugin not found")
 }
 
+func (c *Client) GetLastestPlugin(filter *model.MarketplacePluginFilter) (*model.BaseMarketplacePlugin, error) {
```

Comment:
Nit: should this be `GetLatestPlugin` instead of `GetLastestPlugin`?

Reply:
Good catch, fixed :+1: